src/character.cpp

```diff
@@ -2170,6 +2170,24 @@ ret_val<bool> Character::can_wear( const item &it, bool with_equip_change ) cons
         }
     }
 
+    if( it.has_flag( flag_SPLINT ) ) {
+        bool need_splint = false;
+        for( const body_part bp : all_body_parts ) {
+            if( !it.covers( bp ) ) {
+                continue;
+            }
+            if( is_limb_broken( bp_to_hp( bp ) ) && !worn_with_flag( flag_SPLINT, bp ) ) {
+                need_splint = true;
+                break;
+            }
+        }
+        if( !need_splint ) {
+            return ret_val<bool>::make_failure( ( is_player() ?
+                                                  _( "You don't have any broken limbs this could help." )
+                                                  : string_format( _( "%s doesn't any broken limbs this could help." ), name ) ) );
```

Comment:
```suggestion
                                                  : string_format( _( "%s doesn't have any broken limbs this could help." ), name ) ) );
```